src/components/Profile/Profile.module.css

```diff
@@ -1,16 +1,107 @@
 .wrap {
-    min-width: 300px;
-    padding: 10px;
-    display: flex;
-    border: 1px solid lightgray;   
-  }
-  
-  .avatar {
-    width: 100px;
-    height: auto;
-    padding: 10px;
-  }
-  
-  .text {
-    padding: 10px;
-  }
\ No newline at end of file
+  display: flex;
+  flex-flow: row;
+  justify-content: space-around;
+  align-items: center;
+  width: 700px;
+}
+
+.avatar {
+  width: 180px;
+  height: auto;
+  border-radius: 9px;
+}
+
+.text {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 28px;
+  line-height: 100%;
+  margin: 0;
+  color: #000000;
+}
+
+.description {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 115%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#999999" to be "#999" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/tododo/pullRequest?prid=5504860)